libcontainer/dmz/cloned_binary_linux.go

```diff
@@ -0,0 +1,195 @@
+//go:build linux
+// +build linux
+
```

Comment:
Technically there's no need to add `linux` build tag if the file name ends with `_linux.go`.

Reply:
We have build tag lines in similar files, but sure.